ethereum-types/src/hash.rs

```diff
@@ -59,47 +59,19 @@ macro_rules! impl_uint_conversions {
 	}
 }
 
-impl_serde!(H32, 4);
-impl_serde!(H64, 8);
-impl_serde!(H128, 16);
 impl_serde!(H160, 20);
 impl_serde!(H256, 32);
-impl_serde!(H264, 33);
 impl_serde!(H512, 64);
 impl_serde!(H520, 65);
 
-construct_fixed_hash!{ pub struct H32(4); }
-construct_fixed_hash!{ pub struct H64(8); }
-construct_fixed_hash!{ pub struct H128(16); }
 construct_fixed_hash!{ pub struct H160(20); }
 construct_fixed_hash!{ pub struct H256(32); }
-construct_fixed_hash!{ pub struct H264(33); }
```

Comment:
Used in `parity-ethereum`

Reply:
So we actually can't get rid of any of those hash types.
I will revert the removals to make this no longer break things so that we can merge it.